lib/logstash/inputs/rabbitmq/march_hare.rb

```diff
@@ -112,6 +112,9 @@ def consume
       @consumer = @q.build_consumer(:block => true) do |metadata, data|
         @codec.decode(data) do |event|
           decorate(event)
+	  metadata.headers.each do |header, value|
+            event[header] = value.respond_to?('toString') ? value.toString : value 
+         end if @include_headers
```

Comment:
Effectively it looks like it is indeed an Array, but I was using the RabbitMQ terminology in the management console. 

I was calling .toString because otherwise I was getting a weird hashcode for the headers of type String. Anyways I'll retest/fix this tonight, thanks!